app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,56 @@
+class AlbumsController < ApplicationController
+  def index
+    @media = Album.order(ranking: :desc)
+  end
+
+  def show
+    @media = Album.find(params[:id])
+  end
+
+  def new
+    @media = Album.new
+  end
+
+  def create
+    @media = Album.new(strong_params)
+    @media.ranking = 0
+    if @media.save
+      redirect_to album_path(@media.id)
+    else
+      render "new"
+    end
+  end
+
+  def edit
+    @media = Album.find(params[:id])
+  end
+
+  def update
+    @media = Album.find(params[:id])
+    @media.attributes = strong_params
+    if @media.save
+      render "show"
```

Comment:
What are the trade-offs between using `render "show"` here to directly display the show template, versus redirecting to `albumPath(@media.id)` (as we did in the create action)?